observability/cluster_observability_operator/cluster-observability-operator-release-notes.adoc

```diff
@@ -13,6 +13,30 @@ The {coo-short} complements the built-in monitoring capabilities of {product-tit
 
 These release notes track the development of the {coo-full} in {product-title}.
 
+The following table provides information about which features are available depending on the version of {coo-full} and {product-title}:
+
+[cols="1,1,1,1,1,1,1", options="header"]
+|===
+| COO Version   | OCP Versions       | Distributed tracing   | Logging   | Troubleshooting panel | ACM alerts | Incident detection
+| 1.1+          | 4.12 - 4.14        | ✔                     | ✔         | ✘                     | ✘          | ✘
+| 1.1+          | 4.15               | ✔                     | ✔         | ✘                     | ✔          | ✘
+| 1.1+          | 4.16 - 4.17        | ✔                     | ✔         | ✔                     | ✔          | ✘
+| 1.1+          | 4.18+              | ✔                     | ✔         | ✔                     | ✔          | ✔
+|===
+
+[id="cluster-observability-operator-release-notes-1-1-1_{context}"]
+== {coo-full} 1.1.1
+
+// Back fill advisory when it is created by KOnflux
+// The following advisory is available for {coo-full} 1.1.1:
+//
+// * link:https://access.redhat.com/errata/RHSA-2025:????[RHEA-2025:??? {coo-full} 1.1.1]
+
+
+[id="cluster-observability-operator-1-1-1-bug-fixes_{context}"]
+=== Bug fixes
+
+* Previously, `observability-operator` and `perses-operator` pods entered a `CrashLoopBackOff` state due to `OutOfMemory` errors, after upgrading from {coo-full} 1.0. This release resolves the issue. (link:https://issues.redhat.com/browse/COO-784[*COO-784*])
```

Comment:
```suggestion
* Previously, `observability-operator` and `perses-operator` pods entered a `CrashLoopBackOff` state on many clusters due to `OutOfMemory` errors, after upgrading from {coo-full} 1.0. This release resolves the issue for most cluster configurations.  (link:https://issues.redhat.com/browse/COO-784[*COO-784*])
```

Maybe a bit more careful? We won't be able to set resource limits appropriate to all clusters. Very large clusters will still need to set resource limits in their subscriptions.